src/app/exhibitsPage/page.tsx

```diff
@@ -0,0 +1,83 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+import Link from 'next/link';
+import { IoIosArrowRoundBack } from 'react-icons/io';
+import NavBar from '../../components/userComponents/navBar/navBar';
+import { ExhibitRow } from '../../types/types';
+import { fetchAllExhibits } from '../../supabase/exhibits/queries';
+import Exhibit from '../../components/userComponents/Exhibit/Exhibit';
+
+/**
+ * @param evt on click of button
+ */
+function goBack(evt: React.SyntheticEvent) {
+  // ignore the native anchor action
+  evt.preventDefault();
+
+  window.history.back();
```

Comment:
edge case that can likely be dealt with later: what if this is the first page we end up on? (i.e. if we open up a new tab and go straight to the exhibits page, what should we see?)